<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="API TestSuites...(failed)(failed)"
           tests="3"
           failures="3"
           errors="0"
           skipped="0"
           time="181.690">
  <properties />
            <testcase name="TC_04_GET_EventList_api_check_With_MediaViewer_Positive" time="28.908" classname="com.vbrick.avenger.videocommentapi_AV_9879_testcases.EventListApi_Av_11123">
                <failure type="com.vbrick.Exception.Reasons"
                  message="FAIL"
              >
<![CDATA[com.vbrick.Exception.Reasons: FAIL
  com.vbrick.Exception.CustomReport.checkinglist(CustomReport.java:390)
  com.vbrick.avenger.videocommentapi_AV_9879_testcases.EventListApi_Av_11123.TC_04_GET_EventList_api_check_With_MediaViewer_Positive(EventListApi_Av_11123.java:315)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
            <testcase name="TC_02_GET_EventList_api_check_With_MediaAdmin_Positive" time="27.734" classname="com.vbrick.avenger.videocommentapi_AV_9879_testcases.EventListApi_Av_11123">
                <failure type="com.vbrick.Exception.Reasons"
                  message="FAIL"
              >
<![CDATA[com.vbrick.Exception.Reasons: FAIL
  com.vbrick.Exception.CustomReport.checkinglist(CustomReport.java:390)
  com.vbrick.avenger.videocommentapi_AV_9879_testcases.EventListApi_Av_11123.TC_02_GET_EventList_api_check_With_MediaAdmin_Positive(EventListApi_Av_11123.java:262)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
            <testcase name="TC_01_GET_EventList_api_check_With_AccountAdmin_Positive" time="125.048" classname="com.vbrick.avenger.videocommentapi_AV_9879_testcases.EventListApi_Av_11123">
                <failure type="com.vbrick.Exception.Reasons"
                  message="FAIL"
              >
<![CDATA[com.vbrick.Exception.Reasons: FAIL
  com.vbrick.Exception.CustomReport.checkinglist(CustomReport.java:390)
  com.vbrick.avenger.videocommentapi_AV_9879_testcases.EventListApi_Av_11123.TC_01_GET_EventList_api_check_With_AccountAdmin_Positive(EventListApi_Av_11123.java:228)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
  java.lang.Thread.run(Unknown Source)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
